<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>root_gestor</class>
 <widget class="QMainWindow" name="root_gestor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor - Base de datos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_head">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(116, 6, 144);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>641</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 36pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>GESTOR BASE DE DATOS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_bottom_controller">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>1331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(116, 6, 144);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1311</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="btn_anadir">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>AÃ±adir trabajador</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_mostrar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Mostrar base de datos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_limpiar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Limpiar campos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_borrar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Borrar Registro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_buscar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Buscar Trabajador</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_actualizar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Actualizar base de datos</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_body">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1331</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>411</width>
       <height>504</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_id">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_id"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_fecha_ingreso">
        <property name="text">
         <string>Fecha de Ingreso</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_fecha_ingreso"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_tipo">
        <property name="text">
         <string>Tipo de Trabajador</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txt_tipo">
        <property name="placeholderText">
         <string>Mensual o Jornalero</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_nombre">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txt_nombre"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_apellido">
        <property name="text">
         <string>Apellido</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txt_apellido"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_cedula">
        <property name="text">
         <string>Cedula Nro</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txt_cedula"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbl_valorHora">
        <property name="text">
         <string>Valor de la hora</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txt_valorHora"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbl_hsSimples">
        <property name="text">
         <string>Horas simples</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="txt_hsSimples"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbl_horasExtras">
        <property name="text">
         <string>Horas Extras</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="txt_horasExtras"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lbl_hs_ext_esp">
        <property name="text">
         <string>Horas Extras Especiales</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="txt_hs_ext_esp"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lbl_hsNocturnas">
        <property name="text">
         <string>Horas Nocturnas</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="txt_hsNocturnas"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lbl_jornadaEsp">
        <property name="text">
         <string>Jornada Especial</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="txt_jornadaEsp"/>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="lbl_hijosCargo">
        <property name="text">
         <string>Hijos a Cargo</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="txt_hijosCargo"/>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="lbl_hijosDisca">
        <property name="text">
         <string>Hijos discapacitados</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="lbl_conyugeCargo">
        <property name="text">
         <string>Conyuge a Cargo</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QRadioButton" name="rb_conyuge">
        <property name="text">
         <string>Si</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QLabel" name="lbl_conyugeDisca">
        <property name="text">
         <string>Conyuge Discapacitado</string>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QCheckBox" name="rb_conyugeDisca">
        <property name="text">
         <string>Si</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>0</y>
       <width>921</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
